client/jetpack-cloud/sections/partner-portal/credit-card-fields/credit-card-element-field.tsx

```diff
@@ -19,7 +19,7 @@ export default function CreditCardElementField( {
 	const { formStatus } = useFormStatus();
 	const isDisabled = formStatus !== FormStatus.READY;
 	const { card: cardError } = useSelect(
-		( select ) => ( select( 'credit-card' ) as CreditCardSelectors ).getCardDataErrors(),
+		( select ) => select( creditCardStore ).getCardDataErrors(),
```

Comment:
Nothing special about most of the changed instances - we reference the store through the existing store object instead of its name, which is the recommended way in Gutenberg nowadays.